back/app/services/concerns/side_fx_helper.rb

```diff
@@ -33,4 +33,10 @@ def user_for_activity_on_anonymizable_item(item, user)
   def remove_user_from_past_activities_with_item(item, user)
     user.activities.where(item: item).update_all(user_id: nil)
   end
+
+  def sanitize_change(change)
+    change.delete(:updated_at)
+
+    change
+  end
```

Comment:
The one-liner `change.tap { |h| h.delete(:updated_at) }` doesn't seem any clearer/simpler to me.

Reply:
But maybe we should push responsibility for filtering out this attribute to the FE, so that we can have consistent data structures for `:change` wherever it is added by the BE? 🤔

Reply:
Note: Originally, i was of the opinion the FE should filter out the `updated_at`, but on reflection I realised that we are only adding `object.saved_changes` to the `'changed'` activities selected for the new 'Management feed'. In all other cases where we add a change to the payload, it is for an `attribute.attribute_previous_change`, which doesn't include `updated_at` either.

So, on balance, i feel this is fine for now, and can be adjusted/removed as & when we iterate further on the 'Management feed'.